target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -381,6 +380,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan:2" "1:lan:1" "4:wan" "6@eth0"
 		;;
+	lenovo,newifi-y1s)
+		ucidef_add_switch "switch0" \
+			"5:lan:1" "4:lan:2" "1:lan:3" "2:lan:4" "0:wan" "6@eth0"
```

Comment:
You reversed lan3 and lan4 here.
It should be "1:lan:4" "2:lan:3" "4:lan:2" "5:lan:1" "0:wan" "6@eth0"

Reply:
@981213 
I observed the picture of this device, lan4 is close to wan

see https://image.baidu.com/search/detail?ct=503316480&z=0&ipn=d&word=y1s%20%E8%B7%AF%E7%94%B1%20%E5%9B%BE&step_word=&hs=2&pn=5&spn=0&di=40040&pi=0&rn=1&tn=baiduimagedetail&is=0%2C0&istype=0&ie=utf-8&oe=utf-8&in=&cl=2&lm=-1&st=undefined&cs=1996793438%2C25986711&os=2773541270%2C2348969233&simid=4221212560%2C833948917&adpicid=0&lpn=0&ln=256&fr=&fmq=1565273132438_R&fm=&ic=undefined&s=undefined&hd=undefined&latest=undefined&copyright=undefined&se=&sme=&tab=0&width=undefined&height=undefined&face=undefined&ist=&jit=&cg=&bdtype=0&oriquery=&objurl=http%3A%2F%2Fimg.pconline.com.cn%2Fimages%2Fproduct%2F5784%2F578426%2F3_m2.jpg&fromurl=ippr_z2C%24qAzdH3FAzdH3F2_z%26e3Brv5gstgj_z%26e3Bv54_z%26e3BvgAzdH3Fr6517vpAzdH3Fot6jsjff_657pj6AzdH3Fsjg5e5AzdH3Fc0b9dm_z%26e3Bip4s&gsm=0&rpstart=0&rpnum=0&islist=&querylist=&force=undefined

Reply:
I am really sure about this, maybe need someone help to check...

Reply:
I have this router and I've just checked it before I made the previous comment.

Reply:
The entries of ucidef_add_switch is {port number of switch}:{lan/wan}:{port number on the case} and the order of all entries doesn't matter.

Reply:
lan3 is switch port 2 and lan4 is switch port 1. Your change made lan3 port 1 instead.